_data/meltano/loaders/target-snowflake/meltanolabs.yml

```diff
@@ -93,12 +93,7 @@ settings:
   label: Private Key Passphrase
   name: private_key_passphrase
   sensitive: true
-- description: Private key contents. For KeyPair authentication either `private_key` or `private_key_path` must be provided.
```

Comment:
🤔

Reply:
Yeah, the `pip_url` does point to the PyPI distribution, and the latest there is `0.11.0` which does contain the commit that added the setting so I'm baffled.

Reply:
I'll make a post-release and see that fixes this issue